mergin/merginproject.py

```diff
@@ -154,10 +158,18 @@ def workspace_name(self) -> str:
         return full_name[:slash_index]
 
     def project_id(self) -> str:
-        """Returns ID of the project (UUID using 8-4-4-4-12 formatting without braces)"""
+        """Returns ID of the project (UUID using 8-4-4-4-12 formatting without braces)
+
+        Raises ClientError if project id is not present in the project metadata.
```

Comment:
It is worth adding the note this should only happen with projects downloaded with old client, before February 2023 (#154)

Reply:
Added.